decidim-system/app/views/decidim/system/devise/shared/_links.html.erb

```diff
@@ -1,25 +1,25 @@
 <%- if controller_name != 'sessions' %>
-  <%= link_to "Log in", new_session_path(resource_name) %><br>
+  <%= link_to "Log in", new_session_path(resource_name) %>
 <% end -%>
 
 <%- if devise_mapping.registerable? && controller_name != 'registrations' %>
-  <%= link_to "Sign up", new_registration_path(resource_name) %><br>
+  <%= link_to "Sign up", new_registration_path(resource_name) %>
 <% end -%>
 
 <%- if devise_mapping.recoverable? && controller_name != 'passwords' && controller_name != 'registrations' %>
-  <%= link_to "Forgot your password?", new_password_path(resource_name) %><br>
+  <%= link_to "Forgot your password?", new_password_path(resource_name) %>
 <% end -%>
 
 <%- if devise_mapping.confirmable? && controller_name != 'confirmations' %>
-  <%= link_to "Did not receive confirmation instructions?", new_confirmation_path(resource_name) %><br>
+  <%= link_to "Did not receive confirmation instructions?", new_confirmation_path(resource_name) %>
 <% end -%>
 
 <%- if devise_mapping.lockable? && resource_class.unlock_strategy_enabled?(:email) && controller_name != 'unlocks' %>
-  <%= link_to "Did not receive unlock instructions?", new_unlock_path(resource_name) %><br>
+  <%= link_to "Did not receive unlock instructions?", new_unlock_path(resource_name) %>
 <% end -%>
 
 <%- if devise_mapping.omniauthable? %>
   <%- current_organization.enabled_omniauth_providers.keys.each do |provider| %>
-    <%= link_to "Sign in with #{OmniAuth::Utils.camelize(provider)}", omniauth_authorize_path(resource_name, provider) %><br>
+    <%= link_to "Sign in with #{OmniAuth::Utils.camelize(provider)}", omniauth_authorize_path(resource_name, provider) %>
```

Comment:
This string should be extracted from here. (not in scope of this PR)

Reply:
@alecslupu since an interpolation takes place in the string, how do I call the translation??

Reply:
I am not 100% sure i understand the question ... but i guess you 

```
  #  in your yaml signin_with_provider: "Sign in with %{provider}"
    <%= link_to t(".signin_with_provider", provider: OmniAuth::Utils.camelize(provider)), omniauth_authorize_path(resource_name, provider) %>
```

Reply:
Yes, you answered it. How do I call the i18n string while there is a interpolation taking place in the string. 

I'll give this work around a go.